### PR TITLE
add a section with public keys for validation

### PR DESCRIPTION
**Pull request by Sakurann**
addresses Issue #241.

### CODE REVIEW

**Comment by danielfett** [comment]
The `w3c-vc` example currently uses a different key. We should call that out or change the example to the default key. I think that having one example with a different key (type) doesn't hurt, so we should call out that this example uses a different key.

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 1]
```diff
@@ -1430,6 +1430,30 @@ After the validation, the Verifier will have the following data for further proc
 
 <{{examples/jsonld/verified_contents.json}}
 
+## Elliptic Curve Key Used in the Examples
+
+The following Elliptic Curve public key, represented in JWK format, can be used to validate the Issuer signatures in the above examples (with line wraps within values for display purposes only):
```
Comment (bc-pi) [inline draft-ietf-oauth-selective-disclosure-jwt.md:1435]: This key doesn't have line wraps within values.

```suggestion
The following Elliptic Curve public key, represented in JWK format, can be used to validate the Issuer signatures in the above examples:
```

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 2]
```diff
@@ -1430,6 +1430,30 @@ After the validation, the Verifier will have the following data for further proc
...
+
+```
+{
+  "kty": "EC",
+  "crv": "P-256",
+  "x": "TCAER19Zvu3OHF4j4W4vfSVoHIP1ILilDls7vCeGemc",
+  "y": "ZxjiWWbZMQGHVWKVQ4hbSIirsVfuecCE6t4jT9F2HZQ"
+}
+```
+
```
Comment (bc-pi) [inline draft-ietf-oauth-selective-disclosure-jwt.md:1456]: The holder public keys can already be found in the example content in the `cnf` claim so don't need to be here.
```suggestion

```

**Review by bc-pi — CHANGES_REQUESTED** [review]

**Comment by bc-pi** [comment]
> The `w3c-vc` example currently uses a different key. We should call that out or change the example to the default key. I think that having one example with a different key (type) doesn't hurt, so we should call out that this example uses a different key.

Either way, the holder's public key is already available in the payload of the example SD-JWTs. So it doesn't need to be and shouldn't be included here.

**Review by danielfett — APPROVED** [review]
LGTM with Brian's proposed changes.

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 3]
```diff
@@ -1430,6 +1430,19 @@ After the validation, the Verifier will have the following data for further proc
...
+```
+{
+  "kty": "EC",
+  "crv": "P-256",
+  "x": "b28d4MwZMjw8-00CG4xfnn9SLMVMM19SlqZpVb_uNtQ",
+  "y": "Xv5zWwuoaTgdS6hV43yI6gBwTnjukmFQQnJ_kCxzqk8"
+}
+```
+
```
Comment (Sakurann) [inline draft-ietf-oauth-selective-disclosure-jwt.md:1445]:
```suggestion

The public key used to validate a Key Binding JWT can be found in the examples as the content of the `cnf` claim.

```

**Comment by Sakurann** [comment]
w3c-vc uses a different KB JWT key but the same issuer key, correct?

**Comment by bc-pi** [comment]
> w3c-vc uses a different KB JWT key but the same issuer key, correct?

Yeah, i believe so. But no signed content is actually shown in that example. That example just shows the payload and disclosure content.

**Review by bc-pi — APPROVED** [review]

**Pull request merged**